src/util.jl

```diff
@@ -147,6 +148,17 @@ in the other dimensions.
 function slice(dims::NTuple{N},i,j,low=1) where N
     CartesianIndices(ntuple( k-> k==j ? (i:i) : (low:dims[k]), N))
 end
+using Statistics
+"""
+    removeMean!(s)
+
+Remove the meam of scalar array `s` in physical domain from `s`.
+This is potentially useful for pressure normalization.
+"""
+function removeMean!(s)
+    s̄ = Statistics.mean(@views s[inside(s)])
+    s .-= s̄
+end
```

Comment:
Have you checked on allocations here (I see `@views` is used, but still)? Do we really need `Statistics` to compute the mean? This could be troublesome in GPUs.

Reply:
Yes, @views won't allocate memory and you are right. Statistics.jl is not necessary.